Source/MSBuild.Sdk.Extras/Build/Platforms/Xamarin.targets

```diff
@@ -0,0 +1,28 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+	<PropertyGroup>
+		<MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+	</PropertyGroup>
+
+	<PropertyGroup>
+		<_SdkImplicitPlatformLibraryName>$(TargetFrameworkIdentifier)</_SdkImplicitPlatformLibraryName>
+		<_SdkImplicitPlatformLibraryName Condition="$(_SdkImplicitPlatformLibraryName.StartsWith('Mono'))">$(_SdkImplicitPlatformLibraryName.Replace('Mono', 'Mono.'))</_SdkImplicitPlatformLibraryName>
+	</PropertyGroup>
+
+	<ItemGroup Condition="'$(DisableImplicitFrameworkReferences)' != 'true'">
+		<_SdkImplicitReference Include="System"/>
+		<_SdkImplicitReference Include="System.Runtime"/>
+		<_SdkImplicitReference Include="System.Core"/>
+		<_SdkImplicitReference Include="System.Collections"/>
```

Comment:
One thing to be aware of -- we shouldn't be explicitly referencing these type-forwarder façades. That causes issues since they're added as framework references which then cannot be found in the GAC. We should only be referencing implementation assemblies.

Reply:
I have checked with my projects, no problem there. Besides they are the same references that I added from the original Xamarin project templates.